docs/pages/Configuration.md

```diff
@@ -22,6 +22,7 @@ These options are available and explained in detail below:
 | `decimalSeparator` | Determines the decimal separator for numbers as string.  | string |
 | `hidePreview` | Hide the button to preview the uploaded data in the table dialog.  | boolean |
 | `skipMandatoryFieldCheck` | Skip check if fields with `mandatory` are not filled in spreadsheet file. | boolean |
+| `skipColumnsCheck` | Skip check for unkown columns which are not in the metadata | boolean |
```

Comment:
```diff
- unkown
+ unknown 
```

not worth a PR :P

Reply:
I hope you're just bored to look so deeply in my PRs 😉

Reply:
Nothing negative, just browsing, can always learn from you 😊